### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by Berlin-2206** [livewire/livewire#7806] [1. Help]
Livewire detects a bad function and I don't know how to fix it. Help me: this is the error: Uncaught TypeError: deleteConfirmation is not a function
     onclick http://127.0.0.1:8000/admin/products:1

   <a href="#" onclick="deleteConfirmation({{$product->id}})" class="text-danger">Delete</a>
                                                </td>
                                                
                                            </tr>
                                        @endforeach
                                    </tbody>
                                </table>
                                {{$products->links()}}
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>
    </main>
</div>

<div class="modal" id="deleteConfirmation">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-body pb-30 pt-30">
                <div class="row">
                    <div class="col-md-12 text-center">
                        <h4 class="pb-3">Do you want to delete this record?</h4>
                        <button type="button" class="btn btn-secondary" data-bs-toggle="modal" data-bs-target="#deleteConfirmation">Cancel</button>
                        <button type="button" class="btn btn-danger" onclick="deleteProduct()">Delete</button>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>

@push('scripts')
    <script>
        function deleteConfirmation(id) 
        {
            @this.set('product_id', id);
            $('#deleteConfirmation').modal('show');
        }

        function deleteProduct() 
        {
            @this.call('deleteProduct');
            $('#deleteConfirmation').modal('hide');
        }
    </script>
@endpush

**Comment by jmonday** [comment]
It looks like your formatting was mangled a bit. Can you try reformatting so we can see how things are being called?
